experimental/beacon_sim/make_eigen_bounds_test.cc

```diff
@@ -0,0 +1,75 @@
+#include "experimental/beacon_sim/make_eigen_bounds.hh"
+
+#include "experimental/beacon_sim/test_helpers.hh"
+#include "gtest/gtest.h"
+
+namespace robot::experimental::beacon_sim {
+
+TEST(MakeEigenBoundsTest, step_eigen_bound_no_information) {
+    // Setup 
+    constexpr double start_info_eigen_value = 1.0;
+    constexpr double ub_dynamics_eigen_value = 1.0;
+    constexpr double ub_measurement_eigen_value = 0.0;
+    constexpr double lower_process_noise_eigen_value = 1.0;
```

Comment:
Consider making your test values something other than 0.0 or 1.0, unless you are specifically testing behavior at those values. Adding 0.0 or multiplying by 1.0 are additive and multiplicative identities, so bugs tend to sneak through if you fail to add/multiply something or if you mistakenly do so.